testsuite/integration-tests/pom.xml

```diff
@@ -351,6 +363,87 @@
             <groupId>io.rest-assured</groupId>
             <artifactId>json-path</artifactId>
         </dependency>
+
+        <dependency>
+            <groupId>org.jboss.resteasy</groupId>
+            <artifactId>grpc-bridge</artifactId>
+            <version>${version.resteasy.testsuite}</version>
+        </dependency>
+
+       <dependency>
+            <groupId>jaxrs.example</groupId>
+            <artifactId>jaxrs.example.grpc</artifactId>
+            <version>${version.jaxrs.example.grpc}</version>
+            <type>jar</type>
+            <exclusions>
+               <exclusion>
+                  <groupId>javax.inject</groupId>
+                  <artifactId>javax.inject</artifactId>
+               </exclusion>
+               <exclusion>
+                  <groupId>jaxrs.example</groupId>
+                  <artifactId>jaxrs.example</artifactId>
+               </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+            <groupId>jaxrs.example</groupId>
```

Comment:
We definitely need to change this group id. We have to use something we can claim.

Reply:
Yes, I'll do that.